packages/components/src/templates/next/layouts/Content/Content.tsx

```diff
@@ -136,7 +140,14 @@ const createContentLayoutStyles = tv({
     container:
       "mx-auto grid max-w-screen-xl grid-cols-12 px-6 py-12 md:px-10 md:py-16 lg:gap-6 xl:gap-10",
     siderailContainer: "relative col-span-3 hidden lg:block",
-    content: "col-span-12 flex flex-col gap-16 lg:col-span-9 lg:ml-24",
+    content: "col-span-12 flex flex-col gap-16 lg:col-span-9",
+  },
+  variants: {
+    isSideRailPresent: {
+      true: {
+        content: "lg:ml-24",
```

Comment:
might want to set `lg:col-span-9` here too so content can take up the full 12 columns if siderail is not present, instead of being on the `content` base slot key

Reply:
Ah design wants to keep it to 9 columns only, so the right side is whitespace.

Still pending a decision on what to do with the "Back to top" button, might be putting it on the right side for such pages.

Reply:
Ok decided not to have the "Back to top" button, will leave the right side purely whitespace for now.